index.html

```diff
@@ -2083,22 +1978,20 @@ <h1>References to Existing Security Best Practices</h1>
       and it is important to keep up to date.
       At the same time,
       IoT is new enough and is itself evolving rapidly enough that best practices
-      are just now emerging and are likely to require rapid updating.  
+      are just now emerging and are likely to require rapid updating.
       Recently attempts have been made to document and categorize useful approaches to security in IoT.
       The following are some of the more useful points of reference:
       <ul>
-      <li>[[Garcia17]] - <cite>State-of-the-Art and Challenges for the Internet of Things Security</cite>:<br/>
-      This IETF document, a product of the IETF T2TRG, is still in draft form but is now a candidate for
-      ratification. 
-      It covers a range of recommendations for securing IoT systems.  
+      <li>[[RFC8576]] - <cite>State-of-the-Art and Challenges for the Internet of Things Security</cite>:<br/>
+      This IETF document is a product of the IETF T2TRG and covers a range of recommendations for securing IoT systems.
       Readers should look for the latest version using the IETF RFC tracker.</li>
```

Comment:
No versions anymore since now an RFC.
```suggestion
      </li>
```